README.md

```diff
@@ -227,6 +222,10 @@ For sync communication we have a community slack with a #containerd channel that
 **Slack:** Catch us in the #containerd and #containerd-dev channels on dockercommunity.slack.com.
 [Click here for an invite to docker community slack.](https://join.slack.com/t/dockercommunity/shared_invite/enQtNDY4MDc1Mzc0MzIwLTgxZDBlMmM4ZGEyNDc1N2FkMzlhODJkYmE1YTVkYjM1MDE3ZjAwZjBkOGFlOTJkZjRmZGYzNjYyY2M3ZTUxYzQ)
 
+### Security audit
+
+A third party security audit was performed by Cure53 in 4Q2018; the [full report](docs/SECURITY_AUDIT.pdf) is available in our docs/ directory.
```

Comment:
Are we allowed to put the pdf in the repository (from a licensing perspective?)

Reply:
The CNCF pays for the reports and directs member projects to publish them in their repos, so I'm guessing that has been thought through, but will let @caniszczyk confirm.

Reply:
yes +1 it's CC-BY

Reply:
Thanks! Sorry for the noise ☺️